plugins/transport-nio/src/main/plugin-metadata/plugin-security.policy

```diff
@@ -21,3 +21,18 @@ grant codeBase "${codebase.elasticsearch-nio}" {
   // elasticsearch-nio makes and accepts socket connections
   permission java.net.SocketPermission "*", "accept,connect";
 };
+
+grant codeBase "${codebase.netty-common}" {
+   // for reading the system-wide configuration for the backlog of established sockets
+   permission java.io.FilePermission "/proc/sys/net/core/somaxconn", "read";
```

Comment:
this also looks like it's unneeded since netty catches the exception. Maybe we just omit it?